config/module.go

```diff
@@ -17,6 +17,9 @@ type Module struct {
 	Name string `json:"name"`
 	// ExePath is the path (either absolute, or relative to the working directory) to the executable module file.
 	ExePath string `json:"executable_path"`
+	// Debug represents whether the module executable should be started with -debug. If Debug is unset, the
+	// module will be started with -debug only if the module manager has a Debug log level.
+	Debug *bool `json:"debug"`
```

Comment:
Since you're changing the config structure here, you'll need to also change the protobuf in API for this, and the proto conversion code in config/proto_conversions.go

I believe you'll also need to update App to make sure it delivers the updated structure.

Reply:
Great point; thanks. Looking into other changes, and will merge this PR last (after api and app changes).